SETUP/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ see the git history.
 ## R??????
 Scripts supporting this upgrade are in `SETUP/upgrade/16`
 
+* Remove dependency on jpgraph and GD.
+* Introduct new client rendered graphing capabilities that support theming.
+
```

Comment:
Perhaps:
```
* Removed jpgraph and GD dependencies (chrismiceli)
* Graphs now rendered via client-side Javascript as themed SVGs (chrismiceli)
```

Reply:
Won't this go in `SETUP/upgrade/17`, since the plan is to not include it in the 202109 release?

Reply:
Well handle this in the merge to master i believe unless @cpeel wants me to sync this branch up now?

Reply:
Correct - I'll take care of this on merge to master.